src/app/components/home/add-marios/add-marios.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="container">
+  <div class="back-button-container">
+    <a [routerLink]="['/home']">
+      <button class="back-button" type="button">
+        <img src="assets/icons/left-arrow.svg" alt="" />
+        BACK
+      </button>
+    </a>
+  </div>
+
+  <div class="form-container">
+    <form (ngSubmit)="onSubmit()" #f="ngForm">
+      <ng-select
```

Comment:
Do you need extra library ng-select to use select if it's also in Angular Material UI?

Reply:
Actually I found it much more customizable, with useful built-in features such as text search within binded elements and better UI than mat-select (imo)